lib/lucidboard_web/templates/board/card.html.eex

```diff
@@ -48,7 +48,7 @@
       <p phx-click="inline_edit"
         phx-value="<%= @card.id %>"
         class="content is-size-<%= card_body_size_by_copy(@card.body) %>">
-        <%= @card.body %>
+        <%= @card.body |> String.replace("\n", "<br />") |> raw %>
```

Comment:
The [HTML 5](https://dev.w3.org/html5/html-author/#the-br-element]) doesn't like a closing tag for break anymore and prefers it to be empty.  I had also been in the habit of closing every single tag to adopt to XHTML

Reply:
#makeupyourmind,html